src/metadata_tool/sources/metadata.clj

```diff
@@ -303,6 +304,33 @@
   "The metadata for a specific activity, identified by name."
   (memoize activity-metadata-by-name-fn))
 
+(defn- includes?
+  "Returns true if an element is contained in a list"
+  [list element]
```

Comment:
definitely don't use `list`, a core function, as an argument.
`coll` is better and more standard.